### ISSUE TITLE
TypeError: Cannot read property 'messages' of undefined

### ISSUE THREAD

**Issue by jhabdas** [Adezandee/fast-eslint#173]
<img width="461" alt="screen shot 2016-08-14 at 6 24 27 am" src="https://cloud.githubusercontent.com/assets/440298/17648788/d43b15fe-61e7-11e6-8180-567bea3735f5.png">

Occurs under package version `3.0.0`. To reproduce create an `.eslintrc.js` file with the following contents and attempt to save or modify:

```js
module.exports = {
  env: {
    es6: true,
    node: true
  },
  extends: 'eslint:recommended',
  parserOptions: {
    ecmaVersion: 7,
    sourceType: 'module',
    ecmaFeatures: {
      jsx: true,
      experimentalObjectRestSpread: true,
    }
  },
  plugins: [
    'react'
  ],
  rules: {}
}
```

Despite the message linting continues to function as expected.

**Comment by Adezandee** [comment]
Hello @jhabdas, have you tried to restart Atom after upgrade ? If yes, then can you provide a code sample. Thx

**Comment by jhabdas** [comment]
Minimal steps to reproduce above. For a more involved reproduction:

```
npm i -g react-native-cli
react-native init MyApp
cd MyApp
touch .eslintrc.js
```

And then copy the above code snippet into the ESLint config file, and save the file. As mentioned, the error does not appear to be affecting linting.

**Comment by Adezandee** [comment]
Fixed by #180. Thx for report @jhabdas !

**Comment by jhabdas** [comment]
\o/ yay thanks!!
